tests/test-github-policy.sh

```
@@ -2036,6 +2055,74 @@ if [ "$(id -u)" -ne 0 ]; then
+- Automated tests: pass
+- Manual validation: n/a — none
+$1
+
+## Review tier
+$2
+
+## Why this tier
+contained"
+}
+lr_body '' normal
+accepts && fail "a normal PR without a Local review row was accepted" || ok "no Local review row is refused on normal"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)